spec/acceptance/standalone_spec.rb

```diff
@@ -6,8 +6,7 @@
   let(:puppetdb_params) {}
   let(:puppetdb_master_config_params) {}
 
-  # TODO: temporary work-around for EL installs
-  let(:postgres_version) { "(getvar('facts.os.family') == 'RedHat') ? { true => '12', default => undef }" }
+  let(:postgres_version) { 'undef' } # default
```

Comment:
do we need this let block at all if the value is undef?

Reply:
if the shared example was reworked, no.
the premise is to support testing different versions and/or upgrading in place.